src/game/server/entities/character.cpp

```diff
@@ -1252,18 +1259,18 @@ void CCharacter::Snap(int SnappingClient)
 		pDDNetCharacter->m_Flags |= CHARACTERFLAG_TELEGUN_GRENADE;
 	if(m_Core.m_HasTelegunLaser)
 		pDDNetCharacter->m_Flags |= CHARACTERFLAG_TELEGUN_LASER;
-	if(m_aWeapons[WEAPON_HAMMER].m_Got)
-		pDDNetCharacter->m_Flags |= CHARACTERFLAG_WEAPON_HAMMER;
-	if(m_aWeapons[WEAPON_GUN].m_Got)
-		pDDNetCharacter->m_Flags |= CHARACTERFLAG_WEAPON_GUN;
-	if(m_aWeapons[WEAPON_SHOTGUN].m_Got)
-		pDDNetCharacter->m_Flags |= CHARACTERFLAG_WEAPON_SHOTGUN;
-	if(m_aWeapons[WEAPON_GRENADE].m_Got)
-		pDDNetCharacter->m_Flags |= CHARACTERFLAG_WEAPON_GRENADE;
+	// if(m_aWeapons[WEAPON_HAMMER].m_Got)
+	// 	pDDNetCharacter->m_Flags |= CHARACTERFLAG_WEAPON_HAMMER;
+	// if(m_aWeapons[WEAPON_GUN].m_Got)
+	// 	pDDNetCharacter->m_Flags |= CHARACTERFLAG_WEAPON_GUN;
+	// if(m_aWeapons[WEAPON_SHOTGUN].m_Got)
+	// 	pDDNetCharacter->m_Flags |= CHARACTERFLAG_WEAPON_SHOTGUN;
+	// if(m_aWeapons[WEAPON_GRENADE].m_Got)
+	// 	pDDNetCharacter->m_Flags |= CHARACTERFLAG_WEAPON_GRENADE;
 	if(m_aWeapons[WEAPON_LASER].m_Got)
 		pDDNetCharacter->m_Flags |= CHARACTERFLAG_WEAPON_LASER;
 	if(m_Core.m_ActiveWeapon == WEAPON_NINJA)
```

Comment:
Please remove the whole if and not just the body, otherwise this will become a nested if